src/kajal/assignments/TC11HowManyDpetMangEmp.java

```diff
@@ -0,0 +1,55 @@
+package kajal.assignments;
+
+/*
+ * Test case - 11: [16th Jun'2021]
+ 
+
+program 1: How many Depts we have.
+ans : 4
+
+program 2 : how many managers we have.
+ans : 3
+
+program 3: how many employees we have.
+ans : 8
+ */
+import java.util.List;
+import java.util.TreeSet;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class TC11HowManyDpetMangEmp {
+	static void getDeptName(WebDriver driver) throws InterruptedException {
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method